hypervisor_bridge.c

```diff
@@ -38,6 +39,22 @@
 #include "pcap_capture.h"
 #include "pcap_filter.h"
 
+static int add_nio_desc(nio_t *nio, const char *fmt, ...)
+{
+	int len;
+	va_list argptr;
+
+	va_start(argptr, fmt);
+	len = vsnprintf(NULL, 0, fmt, argptr);
+
+    if (!(nio->desc = malloc((len + 1) * sizeof(char))))
```

Comment:
I'm sorry, but I do not understand what should be freed: i'm not a C programmer...